portal/static/js/main.js

```diff
@@ -86,15 +86,66 @@ var ConsentUIHelper = function(consentItems, userId) {
      * relevant variables currently defined in profile_macro.html, but can be defined by consumer
      *
      */
-    this.ctop = (typeof CONSENT_WITH_TOP_LEVEL_ORG != "undefined") && CONSENT_WITH_TOP_LEVEL_ORG;
-    this.isAdmin = typeof isAdmin != "undefined" && isAdmin ? true: false;
-    this.editable = (typeof consentEditable != "undefined" && consentEditable == true) ? true : false;
-    this.consentDateEditable = this.editable && (typeof isTestPatient != "undefined" && isTestPatient);
+    this.ctop = false;
+    this.isAdmin = false;
```

Comment:
It might make more sense for this to get its value from `tnthAjax.getRoles` (see below), rather than `tnthAjax.getRoles` setting `this.isAdmin`.
```
this.isAdmin = (tnthAjax.getRoles().indexOf("admin") > -1);
```